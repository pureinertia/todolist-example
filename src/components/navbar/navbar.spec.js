import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NavBar from './navbar';

Enzyme.configure({ adapter: new Adapter() });

describe('<NavBar />', () => {
  const props = {
    user: {
      name: 'Some Name',
    },
  };

  it('renders without errors', () => {
    const wrapper = shallow(<NavBar {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
