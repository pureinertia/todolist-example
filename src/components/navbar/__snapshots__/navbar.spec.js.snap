// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> renders without errors 1`] = `
<nav
  className="nav-bar"
>
  <header
    className="nav-bar--header"
  >
    <div
      aria-label="User"
      className="nav-bar--header--photo"
      role="img"
      style={
        Object {
          "backgroundImage": "url([object Object])",
        }
      }
    />
    <h3>
      Some Name
    </h3>
  </header>
  <ul
    className="nav-bar--list"
  >
    <li>
      <a
        className="nav-bar--list--item"
        href="#/"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon="list"
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
        Team To-Do List
      </a>
    </li>
  </ul>
</nav>
`;
